modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectStateLocalServiceImpl.java

```diff
@@ -69,6 +69,15 @@ public void deleteObjectStateFlowObjectStates(long objectStateFlowId) {
 		objectStatePersistence.removeByObjectStateFlowId(objectStateFlowId);
 	}
 
+	@Override
+	public ObjectState fetchObjectStateFlowObjectState(
+			long listTypeEntryId, long objectStateFlowId)
```

Comment:
Did this really require a breaking change commit? This doesn't change the major version of the module